frontend/src/components/dropmenu/DropMenu.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState, useRef } from "react";
+import {
+  makeStyles,
+  Button,
+  ClickAwayListener,
+  Grow,
+  Paper,
+  Popper,
+  MenuItem,
+  MenuList,
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
```

Comment:
let's keep the styles in `.css` file

Reply:
actually to style material ui components using `.css` , I need to write `!important` against every css statement.
![image](https://user-images.githubusercontent.com/60546840/107066488-e92f2800-6792-11eb-8e11-3cdccb4751a3.png)

But materail ui `makeStyles` saves us from that. we dont need to override current Material-ui styles with `!important` when using `makeStyles`.

Reply:
I can style non-material-ui component e.g. `root` class in a separate `.css`. So will push that.

Reply:
@Kajol-Kumari, mam, now i have made a separate `dropmenu.module.css` and I have shifted those stylings of `root` class.